docs/README.adoc

```diff
@@ -0,0 +1,5 @@
+= Contributing to Documentation
```

Comment:
This screams "Contribution.md" and not "Readme.md"

Can you set the readme up like https://github.com/aerogear/aerogear-android-sdk instead with contributing liking to this doc?

Reply:
Here is the android showcase readme too https://github.com/aerogear/android-showcase-template